public_html/wp-content/plugins/pattern-creator/src/components/inspector/settings.js

```diff
@@ -18,15 +31,24 @@ export default function Settings() {
 				<span>Info about saving a block pattern.</span>
 			</div>
 			<PanelBody title="My Block Settings" initialOpen={ true }>
-				<PanelRow>
-					<TextControl label="Pattern Name" value={ title } onChange={ onChange } />
-				</PanelRow>
-				<PanelRow>
-					<TextareaControl label="Description" value={ description } onChange={ onChange } />
-				</PanelRow>
-				<PanelRow>
-					<SelectControl label="Categories" value={ categories } onChange={ onChange } />
-				</PanelRow>
+				<TextControl label="Pattern Name" value={ title } onChange={ onChange } />
+				<TextareaControl label="Description" value={ description } onChange={ setDescription } />
+				<SelectControl
+					label="Pattern Categories"
+					value={ term }
+					options={ termList }
+					onChange={ setTerms }
+				/>
+				<SelectControl
+					label="Preview Width"
+					value={ viewportWidth }
+					onChange={ setViewportWidth }
+					options={ [
+						{ value: 800, label: 'Normal' },
```

Comment:
Should these come from `gutenberg` at some point?

Reply:
They're just placeholder, this whole control will be revamped with the real design in #14